doc/api/crypto.md

```diff
@@ -1365,6 +1365,13 @@ The `key` is the raw key used by the `algorithm` and `iv` is an
 [Buffers][`Buffer`], `TypedArray`, or `DataView`s. If the cipher does not need
 an initialization vector, `iv` may be `null`.
 
+Initialization vectors should be unpredictable; ideally, they will be
+cryptographically random. They do not have to be secret: IVs are typically just
+added to ciphertext messages in plaintext. It may sound contradictory that
+something has to be unpredictable, but doesn’t have to be secret; it’s
```

Comment:
`doesn't` → `does not`, `it's` → `it is`.

Reply:
on it.